tests/test_aiohttp_extract_specs.py

```diff
@@ -0,0 +1,116 @@
+import pytest
+import json
+
+from apistrap.aiohttp import AioHTTPApistrap
+from aiohttp import web
+
+
+async def test_aiohttp_spec_url_repeated_call(aiohttp_client):
+    oapi = AioHTTPApistrap()
+
+    app = web.Application()
+    routes = web.RouteTableDef()
+    oapi.init_app(app)
+
+    @routes.get('/')
+    async def view(request):
+        return web.Response(content_type="application/json", text=json.dumps({
+            "status": "OK"
+        }))
+
+    app.add_routes(routes)
+
+    client = await aiohttp_client(app)
+    response = await client.get('/spec.json')
+
+    assert response.status == 200
+    data = await response.json()
+    assert 'paths' in data
+    assert '/' in data['paths']
+
+    response = await client.get('/spec.json')
+
+    assert response.status == 200
+    data = await response.json()
+    assert 'paths' in data
+    assert '/' in data['paths']
+
+
+async def test_aiohttp_spec_url_weird_method(aiohttp_client):
+    oapi = AioHTTPApistrap()
+
+    app = web.Application()
+    routes = web.RouteTableDef()
+    oapi.init_app(app)
+
+    @routes.route('options', '/')
+    async def view(request):
+        return web.Response(content_type="application/json", text=json.dumps({
+            "status": "OK"
+        }))
+
+    @routes.get('/')
+    async def view(request):
+        return web.Response(content_type="application/json", text=json.dumps({
+            "status": "OK"
+        }))
```

Comment:
Could we put both of these decorators on the first handler function and delete the second.